doc/DEVELOPER.md

```diff
@@ -43,14 +43,25 @@ You can also use `make` instead of `ninja`.
 
 ## Nix Build (Guest Tests + Unit Tests)
 
-To build a Guest Test, run:
+We provide a Nix flake entry (`nix build`) but also a traditional `nix-build`
+entry via `default.nix`.
+
+To simply get all guest tests, run:
+
+```shell
+nix build .#guest-tests
+```
+
+This also builds and runs the unit tests of the CMake project.
+
+To get a specific Guest Test in a specific variant, we have two options:
```

Comment:
You mention two options, but only one is stated. I guess that you mean the flake way of things, but you should state it more explicit.

Reply:
good catch, thanks